### PR TITLE
Provide the ability to use variable message payload in tests

### LINKED ISSUE #96: Send and expect custom payload in tests
**Problem to Solve**
Custom payload in tests.

**Possible Solution**
Parse metadata from `.meta.wasm` and encode/decode data provided from `.yaml` with SCALE.

**Notes**
Something like this:
```yaml
title: meta

programs:
  - id: 1
    path: examples/target/wasm32-unknown-unknown/release/demo_meta.wasm
    meta: examples/target/wasm32-unknown-unknown/release/demo_meta.meta.wasm
    init_message:
      kind: custom
      value:
        value: 1
        annotation: "hello"

fixtures:
  - title: meta
    messages:
      - destination: 1
        payload:
          kind: custom
          value:
            value: 1
            annotation: "hello"
    expected:
      - messages:
          - destination: 1
            payload:
              kind: custom
              value:
                old_value: 0
                new_value: 1
```

### PR DESCRIPTION
**Pull request by breathx**
Fixes #96.

@gear-tech/dev

### CODE REVIEW

**Comment by NikVolf** [comment]
what is this? substrate update?

File: gstd/src/msg.rs [hunk 1]
```diff
@@ -56,6 +57,10 @@ pub fn load_bytes() -> Vec<u8> {
     result
 }
 
+pub fn load_custom<D: DeserializeOwned>() -> serde_json::Result<D> {
```
Comment (NikVolf) [inline gstd/src/msg.rs:60]: this is just non-sense, we don't need any serde/json in `gstd`

**Comment by breathx** [comment]
@NikVolf 
Diff in '.lock' after merging PR#176

**Comment by NikVolf** [comment]
> @NikVolf
> Diff in '.lock' after merging #174

merge master to get rid of it

**Comment by breathx** [comment]
The feature will be implemented in a different way in another pull request

**Comment by breathx** [comment]
The feature will be implemented in a different way in another pull request.

**Pull request closed without merging**